domains/client/domain-executor/src/bundle_producer.rs

```diff
@@ -252,10 +252,10 @@ where
                 transcript_data.clone(),
             ) {
                 // TODO: specify domain_id properly.
-                const SYSTEM_DOMAIN_ID: u64 = 0;
+                const SYSTEM_DOMAIN_ID: u32 = 0;
```

Comment:
This could have been `DomainId::SYSTEM` constant, just like there are `u32::MAX` and such